packages/DialogActions/src/DialogActions.js

```diff
@@ -0,0 +1,112 @@
+import React from "react";
+import PropTypes from "prop-types";
+import L10n from "@paprika/l10n";
+import useI18n from "@paprika/l10n/lib/useI18n";
+import Button from "../../Button/src/Button";
+import ButtonStyles from "./DialogActions.styles";
+
+const propTypes = {
+  /** Optional custom classes. */
+  className: PropTypes.string,
+
+  /** If the primary button is a destructive action. */
+  confirmKind: PropTypes.string,
```

Comment:
shouldn't this be `kindConfirm`? following kind of the same of structure like defaultValue,